cartoframes/auth/__init__.py

```diff
@@ -6,7 +6,7 @@
 _default_context = None
 
 
-def set_default_context(context=None, base_url=None, api_key=None, creds=None, session=None):
+def set_default_context(context=None, base_url=None, api_key='default_public', creds=None, session=None):
```

Comment:
It feels a little strange to me that `context` can be used for a `base_url` or a `cartoframes.Context` object, but it also makes some logical sense that a base url is essentially a context if there's a default api key.

Some ideas to get around this:
+ force keyword args using the `*` notation in the function signature, and let the first arg be a more generic thing
+ rename the `context` kwarg?
+ do nothing because it's fine :) But clear docs explaining what's going on

Reply:
Yes. The first param can be a context or a base_url. Renaming it to be more generic is a good approach to me.